ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXTerrainPlanningDebugger.java

```diff
@@ -204,7 +204,7 @@ public void addExpansionSpheresFromActionSet(FramePose3D footstepPose, RobotSide
          action.scale(1 / 50.0f);
          spherePosition.add(action);
 
-         float height = terrainMapData.getHeightInWorld((float) spherePosition.getX(), (float) spherePosition.getY());
+         double height = terrainMapData.getHeightInWorld((float) spherePosition.getX(), (float) spherePosition.getY());
```

Comment:
Kinda pedantic but these casts aren't needed anymore
```suggestion
         double height = terrainMapData.getHeightInWorld(spherePosition.getX(), spherePosition.getY());
```